dpnp/backend/extensions/statistics/sliding_window1d.hpp

```diff
@@ -574,9 +574,18 @@ void submit_sliding_window1d(const PaddedSpan<const T, SizeT> &a,
             }
 
             auto *const out_ptr = out.begin();
-            auto *const out_end = out.end();
-            results.store(&out_ptr[glid],
-                          [out_end](auto &&ptr) { return ptr < out_end; });
+            // auto *const out_end = out.end();
+
+            auto y_start = glid;
+            auto y_stop =
+                std::min(y_start + WorkPI * results.size_x(), out.size());
+            int32_t i = 0;
```

Comment:
Shouldn't it be unsigned?
```suggestion
            uint32_t i = 0;
```

Reply:
Done